### PR TITLE
Add Sinhala translation

### PR DESCRIPTION
**Pull request by Thisuu**

### CODE REVIEW

File: svelte/src/libs/translations/languages/si.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+  "id": "en",
```
Comment (getneil) [inline svelte/src/libs/translations/languages/si.json:2]: should not be en, si?

Comment (Thisuu) [reply svelte/src/libs/translations/languages/si.json]: Oh, missed it somehow :)

**Comment by Thisuu** [comment]
Any errors to be fixed here?

**Comment by getneil** [comment]
> Any errors to be fixed here?

there are conflicts. check this pr from time to time.

**Comment by getneil** [comment]
thanks @Thisuu

**Pull request merged**